instrument/OSIRIS_Definition.xml

```diff
@@ -760,6 +760,32 @@
   <algebra val="app-shape" />
 </type>
 
+  <component type="silica" idlist="silica">
```

Comment:
Can you confirm that this should indeed be `silica` in this file, not `silicon`?

Reply:
@MialLewis @AnthonyLim23 it should be silicon